pkgs/by-name/on/onscripter-en/package.nix

```
@@ -0,0 +1,61 @@
+{
+  SDL,
+  SDL_image,
+  SDL_mixer,
+  SDL_ttf,
+  fetchFromGitHub,
+  freetype,
+  lib,
+  libjpeg,
+  libogg,
+  libpng,
+  libvorbis,
+  pkg-config,
+  smpeg,
+  stdenv,
+  libx11,
+}:
+
+stdenv.mkDerivation {
+  pname = "onscripter-en";
+  version = "2025-06-08";
+
+  src = fetchFromGitHub {
+    owner = "Galladite27";
+    repo = "ONScripter-EN";
+    rev = "https://github.com/Galladite27/ONScripter-EN/releases/tag/2025-06-08";
```

you know this'll generate a link akin to
```
https://github.com/Galladite27/ONScripter-EN/archive/refs/tags/https://github.com/Galladite27/ONScripter-EN/releases/tag/2025-06-08.tar.gz
```
?
```suggestion
    tag = "2025-06-08";
```
THIS is what a tag looks like